src/neo-vm/ExecutionEngine.cs

```diff
@@ -62,6 +62,13 @@ public class ExecutionEngine : IDisposable
         public ExecutionContext EntryContext => InvocationStack.Count > 0 ? InvocationStack.Peek(InvocationStack.Count - 1) : null;
         public VMState State { get; internal protected set; } = VMState.BREAK;
 
+        #region Events
+        
+        public event EventHandler<ExecutionContext> ContextLoaded;
```

Comment:
For events, I prefer to add `On` before the name

Reply:
It is no longer popular to add `On` before event names.